### ISSUE TITLE
Missing wdio.cmd on Windows

### ISSUE THREAD

**Issue by liabru** [webdriverio/gulp-webdriver#32]
The line:

```
wdioBin = require.resolve(path.join('webdriverio', 'bin', isWin ? 'wdio.cmd' : 'wdio'))
```

Fails because I'm missing `node_modules\webdriverio\bin\wdio.cmd` after doing a fresh install. The only file in `bin` is `wdio` but this won't run on Windows.

I suspect the issue is due to the change in d7aca92 because `node_modules/.bin/wdio.cmd` does actually exist.

The weird thing is version `1.0.1` doesn't seem to work either now, I get the error message `Haven't found the WebdriverIO test runner` even though `node_modules/.bin/wdio.cmd` does actually exist.

Windows 10, npm 3.3.12, node v5.3.0, webdriverio 3.4.0, gulp-webdriver 1.0.2

**Update**: I got it working by changing the line to:

```
wdioBin = path.join(__dirname, '..', '.bin', isWin ? 'wdio.cmd' : 'wdio')
```

I'm guessing this is a problem due to the new npm flat dependency system?

**Comment by mbroadst** [comment]
+1 seeing this as well

**Comment by joao-arvins** [comment]
+1

**Comment by gilmarsquinelato** [comment]
This was fixed [#35](https://github.com/webdriverio/gulp-webdriver/pull/35)

**Comment by christian-bromann** [comment]
Also fixed with `v2.0.0`

**Comment by liabru** [comment]
Thanks guys!
